tests/OmniSharp.Roslyn.CSharp.Tests/TypeLookupFacts.cs

```diff
@@ -499,7 +499,7 @@ public class TestClass
 }";
             var response = await GetTypeLookUpResponse(content);
             var expected =
-            @"This sample shows how to call the TestClass.GetZero method.
+@"This sample shows how to call the TestClass.GetZero  method.
```

Comment:
Why are we adding extra whitespace here?

Reply:
Initially `return cref + " ";` in GetCref was appending a space after `TestClass.GetZero` and the space before `method` was being trimmed so overall there was a single space. Now we are retaining the space before `method` as well, hence the output consists of 2 spaces. But this shouldn't be an issue as vscode collapses the multiple spaces in the text (as Hover API uses a MarkDown string)